creditrisk/models/predict.py

```
@@ -232,102 +231,103 @@ def predict(
+
     if model_info is None:
         logger.info("No champion model found, checking for any models...")
         try:
             registered_models = client.search_registered_models()
             if not registered_models:
                 logger.info("No models found in registry. Please train a model first.")
-                exit(0)
+                sys.exit(0)
             model_info = registered_models[0].latest_versions[0]
             logger.info(f"Using latest model version: {model_info.version}")
         except MlflowException as e:
```

Ruff (0.8.2)

Undefined name `MlflowException`

(F821)